.github/ISSUE_TEMPLATE/bug_template.yml

```diff
@@ -0,0 +1,86 @@
+name: 🐛 Bug report
+description: Create a report to help us improve
+title: "[BUG] <title>"
+labels: ['bug, untriaged']
+body:
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Related component
+      description: Choose a specific OpenSearch component your bug belongs to.  If you are unsure which to select or if the component is not present, select "Other".
+      multiple: false
+      options:
+        - Build
```

Comment:
This is a tricky, Since "Build" component is the first option and selected by default. Is there away to not be selected by default. Or make Other as the default option.
Or have something like this 
![Screenshot 2023-12-08 at 11 09 40 AM](https://github.com/opensearch-project/OpenSearch/assets/25317259/7cab11dd-e47c-4e47-81fb-c4ee02c572a0)

Reply:
Was looking at that but didn't see a way to do that yet with GH forms - https://github.com/orgs/community/discussions/61648

Reply:
updated to put "Other" first... not perfect but at least if user doesn't make any changes we aren't setting as build.